script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -496,12 +497,13 @@ The tail lamps of the passing cars.
 [sha:fc7181efba60a3670f85abe61c1e57dc7acfc619]{
```

Comment:
github won't let me comment, but 20487 should be "The brake lights of passing cars."